executor/executor_windows.h

```diff
@@ -6,6 +6,9 @@
 
 #include "nocover.h"
 
+#define read read_win
+#define write write_win
```

Comment:
I think read_win/write_win functions should be moved to this file as well. They are only useful with these defines.

Reply:
Since read and write are commonly used I thought they'd go into the common_* file. Sure thing I will move the functions here.

Reply:
The common*.h files are shared between executor and generated C reproducers. If these functions are not used in C reproducers (and they don't), then the functions should go into executor* files.
Unused functions in C reproducers will cause test failures as well (when we get to testing windows support).

Reply:
Oh okay I've moved the functions from common* to executor_*.